mediatimestamp/__init__.py

```diff
@@ -130,6 +137,9 @@ class TimeOffset(object):
     ROUND_NEAREST = 1
     ROUND_UP = 2
 
+    MAX_NANOSEC = 1000000000
```

Comment:
Should these be assigned using the global variables while they still exist?

Reply:
My intention was not to, so that removing the global variables later would be easier.

A case could be made that the global variables should be assigned using these, maybe.